golang/connection.go

```diff
@@ -552,7 +553,11 @@ func (c *Connection) SendSystemError(id uint32, span *Span, err error) error {
 
 // connectionError handles a connection level error
 func (c *Connection) connectionError(err error) error {
-	c.log.Warnf("Connection error: %v", err)
+	if err == io.EOF {
+		c.log.Debugf("Connection got EOF")
```

Comment:
Yep, just means connection closed. This happens a lot when using tcurl for health checks - since the tcurl process dies, and so it's expected.